CreateAnalyticsServiceServiceDbTables.sql

```diff
@@ -1,3 +1,6 @@
+USE [AnalyticsServiceServiceDb]
```

Comment:
Oh, I didn't know that. I just run it locally and forgot to change DB, so my tables were created in the wrong one. I'll create another script, just for local development and PR it :)